public-servers.json

```diff
@@ -36,12 +35,14 @@
 			"location": "USA",
 			"host": "server.cockatrice.us",
 			"port": 4747,
+			"websocketPort": 4748,
 			"authRequired": true,
 			"regRequired": true
 		},
 		{
 			"id": 4,
 			"name": "Woogerworks",
+			"site": https://woogerworks.com/,
```

Comment:
Any clue why the URL is highlighted in red? Is it only me?

Reply:
its not in quotes